.php-cs-fixer.dist.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+declare(strict_types = 1);
```

Comment:
how is this related to linked fabbot failure ?

```
diff -ru tests/DocBlock/AnnotationTest.php tests/DocBlock/AnnotationTest.php
--- tests/DocBlock/AnnotationTest.php	2021-12-16 12:23:15.117358310 +0000
+++ tests/DocBlock/AnnotationTest.php	2021-12-16 12:23:17.200274487 +0000
@@ -503,7 +503,7 @@
         $this->expectException(\RuntimeException::class);
         $this->expectExceptionMessage('This tag does not support types');
 
-        $tag = new Annotation([new Line(' * @deprecated since 1.2')]);
+        $tag = new Annotation([new Line(' * @deprecated since Symfony 1.2')]);
 
         $tag->getTypes();
     }
```

Reply:
I wanted to modify each file, so fabbot complains about everything it would in the repo, fix it and updated config back, so all `declare` chase are reverted and only fabbot's stays.

Reply:
got you! cool idea ;) maybe next time leave a heads up, so we don't wonder what's going on :D